src/main/java/io/fusionauth/client/FusionAuthClient.java

```diff
@@ -4363,6 +4379,29 @@ public ClientResponse<LoginReportResponse, Errors> retrieveUserLoginReportByLogi
         .go();
   }
 
+  /**
+   * Retrieves the login report between the two instants for a particular user by login Id, using specific loginIdTypes. If you specify an application id, it will only return the
+   * login counts for that application.
+   *
+   * @param applicationId (Optional) The application id.
+   * @param loginId The userId id.
+   * @param start The start instant as UTC milliseconds since Epoch.
+   * @param end The end instant as UTC milliseconds since Epoch.
+   * @param loginIdTypes the identity types that FusionAuth will compare the loginId to. Defaults to [email, username]
```

Comment:
This is a bit weird that we state defaults here, even though this is a separate overload from where `loginIdTypes` is truly "optional". But it shows up here since we are sharing the API json files with languages that do not support overloading.

If it's a big deal, could probably add more FTL code that slaps the defaults on the end of the param comment only in those languages.